### ISSUE TITLE
Feature request: filter out movies in Trakt history

### ISSUE THREAD

**Issue by eRJe79** [josh/imdb-trakt-sync#15]
HI,
Thanks for these set of scripts.

In Trakt when a movies has been watched, Trakt adds this to the history and removed the movie from the watch list. This is a nice way to keep the watch list clean. But now when the script is being invoked, IMDb will populate the Trakt watch list with the same movies again.

It would be nice to have an option to filter out movies that occur in the Trakt history before syncing the IMDb list with Trakt.

**Comment by josh** [comment]
> IMDb will populate the Trakt watch list with the same movies again.

Yeah it's a bummer I can't figure out a good way to do true 2 way sync here. Ideally I'd want removing something from Trakt watched to remove it from IMDb. But the way you kinda have to deal with it is to treat IMDb as the canonical source.

> It would be nice to have an option to filter out movies that occur in the Trakt history before syncing the IMDb list with Trakt.

I probably can't implement exactly what you're describing but wondering if it would be helpful to be able to just disable the watchlist sync entirely and just sync say ratings and history?

https://github.com/josh/imdb-trakt-sync/blob/c0363ddea16878cfe6bcde2d85b0394a102a272b/sync.sh#L18-L22

Basically we'd just skip running lines 18 and 19 if you set some docker environment flag.
